<div class="chatlog__message-group">
  <div id="chatlog__message-container-454476755256999946" class="chatlog__message-container " data-message-id="454476755256999946">
    <div class="chatlog__message">
      <div class="chatlog__message-aside">
        <img class="chatlog__avatar" src="https://cdn.discordapp.com/avatars/293624673265123328/accd07acc220a18568ba46a6e9ede18a.png?size=512" alt="Avatar" loading="lazy">
      </div>
      <div class="chatlog__message-primary">
        <div class="chatlog__header">
          <span class="chatlog__author" style="color: rgb(17, 128, 106)" title="koher#9336" data-user-id="293624673265123328">koher</span>
          <a href="/channels/453733491067322378?category=main&amp;channel=apple-dev&amp;message_id=454476755256999946"><span class="chatlog__timestamp">08-Jun-18 02:48 AM</span></a>
        </div>
        <div class="chatlog__content chatlog__markdown">
          <span class="chatlog__markdown-preserve"><span class="chatlog__markdown-mention" title="hironytic#5238">@hironytic</span>
            <div class="chatlog__markdown-quote">
              <div class="chatlog__markdown-quote-border"></div>
              <div class="chatlog__markdown-quote-content">あまりわかってませんが、 init(contentsOf:) がthrowsなんなら、 Bundleから読み込む init() のときだけLogic failureなのは辻褄あってない気がしますが。 </div>
            </div> 一般論として、 <span class="chatlog__markdown-pre chatlog__markdown-pre--inline">foo() throws</span> を用いる関数 <span class="chatlog__markdown-pre chatlog__markdown-pre--inline">bar()</span> が <span class="chatlog__markdown-pre chatlog__markdown-pre--inline">foo() throws</span> がエラーを <span class="chatlog__markdown-pre chatlog__markdown-pre--inline">throw</span> するようなケースを Logic failure とする事前条件を持つことはありえると思います。
          </span>
          <span class="chatlog__edited-timestamp" title="08-Jun-18 02:48 AM">(edited)</span>
        </div>
      </div>
    </div>
  </div>
</div>